.github/workflows/e2e.yaml

```diff
@@ -11,6 +11,12 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+
+      - name: Run Hatchet Engine
+        uses: hoverkraft-tech/compose-action@v2.0.1
```

Comment:
Generally against third-party Github actions unless they're from a very trusted source -- seems like this could be a single command?

Reply:
works for me!

Reply:
fixed that 👍